demisto_sdk/commands/init/tests/contribution_converter_test.py

```diff
@@ -871,7 +871,11 @@ def test_rearranging_before_conversion_indicator_and_incident_fields(repo: Repo)
     "input_script, output_version",
     [
         (
-            "This is a test script\n the script contains a pack version\n ### pack version: 3.4.5  TEST TEST",
```

Comment:
We should still support this type of pack version... Lets keep that test as well.